app/Resources/views/press-pack.html.twig

```diff
@@ -26,6 +26,12 @@
                 {{ render_pattern(block) }}
             {% endfor %}
 
+            {% if is_granted('FEATURE_CAN_USE_HYPOTHESIS') %}
```

Comment:
since `hypothesisOpener` is passed in by the controller only if `FEATURE_CAN_USE_HYPOTHESIS` is granted, perhaps we can only check for its existence here. `is defined` does that https://github.com/elifesciences/journal/blob/9eee7beb02769c38320219bd444774013b04552d/app/Resources/views/article.ris.twig#L30